Swabbr.Core/Helpers/TimeZoneInfoHelper.cs

```diff
@@ -26,11 +28,51 @@ public static string MapTimeZoneToStringOrNull(TimeZoneInfo timeZoneInfo)
             }
 
             var offset = timeZoneInfo.BaseUtcOffset;
+            var absHours = Math.Abs(offset.Hours);
 
-            var hours = (offset.Hours >= 10) ? $"{offset.Hours}" : $"0{offset.Hours}";
-            var minutes = (offset.Minutes >= 10) ? $"{offset.Minutes}" : $"0{offset.Minutes}";
+            var hours = (absHours >= 10) 
```

Comment:
This used to be the case for `string.Format(..)` but I am not sure that's any better than this.